src/views/waypoint/home/components/get-started/style.module.css

```diff
@@ -0,0 +1,81 @@
+.root {
+  overflow: hidden;
+  margin-bottom: 48px;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  margin: -8px;
+}
+
+.iconSection {
+  display: block;
+  flex: 0 0 auto;
+  margin: 8px;
+
+  & svg {
+    display: block;
+    height: auto;
+    width: 96px;
+  }
+}
+
+.textSection {
+  flex: 1 1 0;
+  margin: 8px;
+  min-width: 400px;
+}
+
+.heading {
+  color: var(--token-color-palette-neutral-700);
+  /**
+    * NOTE: !important is necessary here, as higher specificity styles
+    * are bleeding in from docs layout styles. One path to resolve this
+    * might be to use MDX custom components, rather than raw-element-targeting styles,
+    * in order to styles docs contents
+    **/
```

Comment:
Agree with this, I think a good first test will be with the headings and permalink icons!